sympy/solvers/ode.py

```diff
@@ -3028,20 +3028,29 @@ def _handle_Integral(expr, func, hint):
         sol = expr
     return sol
 
-def _ode_factorable_match(eq, func):
+def _ode_factorable_match(eq, func, x0):
 
     from sympy.polys.polytools import factor
     eqs = factor(eq)
     eqs = fraction(eqs)[0] # p/q =0, So we need to solve only p=0
     eqns = []
     r = None
+    if isinstance(eqs, Pow):
+        eq, _ = _preprocess(eqs, func)
```

Comment:
Are we calling `_preprocess` just so it can do:
```julia
if eq.exp.is_positive:
    eq = eq.base
```
If so that seems like overkill and we should just have that code here.